quic/s2n-quic/src/provider/datagram.rs

```diff
@@ -3,6 +3,7 @@
 
 //! Provides unreliable datagram support
 
+pub use s2n_quic_core::datagram::default::{Endpoint as DefaultEndpoint, Sender};
```

Comment:
I would just do this and call it a day :)

```suggestion
pub use s2n_quic_core::datagram::default;
```

and then when you go to import it elsewhere you can rename it however you'd like.